packages/eslint-plugin-lit-a11y/tests/lib/rules/valid-lang.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileoverview Ensures the document has a valid `lang` attribute.
+ * @author open-wc
+ */
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const { RuleTester } = require('eslint');
+const rule = require('../../../lib/rules/valid-lang');
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+  settings: { litHtmlSources: false },
+  parserOptions: {
+    sourceType: 'module',
+    ecmaVersion: 2015,
+  },
+});
+
+ruleTester.run('valid-lang', rule, {
+  valid: [
```

Comment:
I've only added a few tests here, please let me know if you feel I can add some more cases

Reply:
Maybe we can add some test cases to see how well it handles expressions in templates, e.g.:
```
html`<html lang=${foo}></html>`
html`<html lang=${'bar'}></html>`
html`<html lang=${1}></html>`
```

Reply:
Great shout, this brought to light a few issues, and they've been fixed!